app/component/StopNearYouContainer.js

```diff
@@ -126,6 +127,11 @@ const containerComponent = createRefetchContainer(
         ...StopNearYouContainer_stop
         @arguments(startTime: $startTime, omitNonPickups: $omitNonPickups)
       }
+
+      station(id: $stopId) {
```

Comment:
I didn't test this yet but does the code somehow magically already handle that sometimes stations is now returned?

Reply:
Yes, it seems that the code already handles this. All the stops and stations seem to work like before without changes to code